ArduPlane/Parameters.h

```diff
@@ -542,6 +543,7 @@ class ParametersG2 {
     AP_Gripper gripper;
 #endif
 
+    AP_LandingGear landinggear;
```

Comment:
tiny change, mind calling this landing_gear?

Reply:
fixed